edward/util/random_variables.py

```diff
@@ -712,3 +714,57 @@ def get_variables(x, collection=None):
     nodes.update(node.op.inputs)
 
   return list(output)
+
+
+def transform(x, *args, **kwargs):
+  """Transform a continuous random variable to be on the unconstrained
+  space. It selects according to a number of default transformations,
+  depending on the support.
+
+  Parameters
```

Comment:
Our docstring format changed to follow TensorFlow's. Can you make that change? (c.f., the other function docstrings in this file)